solo-chains/runtime/dancelight/src/xcm_config.rs

```diff
@@ -116,6 +117,23 @@ pub type XcmRouter = WithUniqueTopic<
     ChildParachainRouter<Runtime, XcmPallet, PriceForChildParachainDelivery>,
 >;
 
+/// Type for specifying how a `Location` can be converted into an `AccountId`. This is used
+/// when determining ownership of accounts for asset transacting and when attempting to use XCM
+/// `Transact` in order to determine the dispatch Origin.
+pub type LocationToAccountId = (
```

Comment:
I guess we should remove this. For dancelight, we already have the proper `LocationConverter` configured.

See: https://github.com/moondance-labs/tanssi/blob/master/solo-chains/runtime/dancelight/src/xcm_config.rs#L64